src/main/scala/gremlin/scala/GremlinScala.scala

```diff
@@ -351,6 +351,9 @@ object GremlinScala {
 
     def hasNot(key: String, value: Any) = GremlinScala[End, Labels](traversal.where(P.not(__.has[End](key, value))))
 
+    def and(traversals: GremlinScala[End, HNil]*) =
+      GremlinScala[End, Labels](traversal.and(traversals.map(_.traversal):_*))
```

Comment:
Given I don't have much experience with `HNil` I wasn't sure about typing here, just tried to follow other steps. Could it be better perhaps?